<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollabDialog</class>
 <widget class="QDialog" name="CollabDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collaborate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="sessionType">
     <property name="styleSheet">
      <string notr="true">QFrame {
     border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QPushButton" name="clientButton">
        <property name="text">
         <string>Join</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="serverButton">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="selSession">
        <property name="text">
         <string>Select collaboration session type:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="collabInfo">
     <property name="styleSheet">
      <string notr="true">QFrame {
    border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="collabNick"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="collabNickLabel">
        <property name="text">
         <string>Nick</string>
        </property>
        <property name="buddy">
         <cstring>collabNick</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="collabAddrLabel">
        <property name="text">
         <string>Address</string>
        </property>
        <property name="buddy">
         <cstring>collabAddr</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="collabAddr"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="collabPortLabel">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>collabPort</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="collabPort"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientButton</tabstop>
  <tabstop>serverButton</tabstop>
  <tabstop>collabNick</tabstop>
  <tabstop>collabAddr</tabstop>
  <tabstop>collabPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
